django/thunderstore/core/urls.py

```diff
@@ -27,12 +28,14 @@
 AUTH_ROOT = "auth/"
 
 urlpatterns = [
-    path("", PackageListView.as_view(), name="index"),
+    path("", CommunitiesView.as_view(), name="index"),
     path("ads.txt", ads_txt_view, name="ads.txt"),
     path("robots.txt", robots_txt_view, name="robots.txt"),
     path(AUTH_ROOT, include("social_django.urls", namespace="social")),
     path("logout/", LogoutView.as_view(), kwargs={"next_page": "/"}, name="logout"),
-    path("package/", include(repository_urls)),
+    path("package/", include(old_urlpatterns)),
+    path("communities/", CommunitiesView.as_view(), name="communities"),
+    path("c/", include(community_urls)),
```

Comment:
Define namespace for these patterns to help url reversing

Reply:
I removed the names from legacy package urls. Since those should be redirected to in the future.

I didn't want to put a namespace on the community or communities, because then all the package url names should be prefixed with `community:package.create` I'm not so certain that's what we want to do.

but if we want to let me know 👍